manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -354,7 +354,7 @@ spec:
               name: vsphere-config-volume
               readOnly: true
         - name: csi-provisioner
-          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.1.1
```

Comment:
The latest is 3.2.0 and 3.2.0 is dependent on external-snapshotter 6.0.1.

Reply:
If we update csi-snapshotter sidecar to 6.0.1, we should also make sure snapshot CRDs, snapshot-controller, snapshot validation webhook are also installed from 6.0.1.
cc @deepakkinni